nemo_curator/scripts/semdedup/clustering.py

```diff
@@ -46,15 +45,11 @@ def main(args):
     )
 
     client = get_client(**ArgumentHelper.parse_client_args(args))
-    dt1 = datetime.now()
+    dt1 = datetime.now()  # noqa: DTZ005
```

Comment:
Instead of `datetime.now()` it could be `time.perf_counter()` like elsewhere in this PR?